Cargo.toml

```
@@ -1,7 +1,7 @@
 [package]
 name = "uad_gui"
 description = "A cross-platform GUI debloater for android devices"
-version = "0.6.1"
+version = "0.6.2"
 authors = ["Universal-Debloater-Alliance"]
 license = "GPL-3.0"
 homepage = "https://github.com/Frigyes06/universal-android-debloater"
```

Owner has changes should `homepage`, and `repository` be updated to the https://github.com/Universal-Debloater-Alliance/universal-android-debloater right?
```suggestion
homepage = "https://github.com/Universal-Debloater-Alliance/universal-android-debloater"
repository = "https://github.com/Universal-Debloater-Alliance/universal-android-debloater"
```

---

Well spotted, thanks.